docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -562,26 +569,103 @@
       }
     },
     "extensions": {
-      "type": "array",
-      "description": "List of link and component extensions which the plugin registers to given extension points.",
-      "items": {
-        "type": "object",
-        "properties": {
-          "extensionPointId": {
-            "type": "string"
-          },
-          "title": {
-            "type": "string"
-          },
-          "description": {
-            "type": "string"
-          },
-          "type": {
-            "type": "string",
-            "enum": ["link", "component"]
+      "type": "object",
+      "description": "...",
+      "properties": {
+        "addedComponents": {
+          "type": "array",
+          "description": "...",
+          "items": {
+            "type": "object",
+            "properties": {
+              "targets": {
+                "type": "array",
+                "description": "...",
+                "items": {
+                  "type": "string"
+                }
+              },
+              "title": {
+                "type": "string",
+                "description": "..."
```

Comment:
If we want to use `title` to associate issues that we display in the admin page, we may want to set a `minLength` on this field.

Reply:
Good point 👍

Reply:
Addressed in https://github.com/grafana/grafana/pull/92732/commits/26969323c4db152e11b5079fe90621ba0303b9fa. Do you think `10` characters would be ok?

Reply:
Sounds good to me. 👍

Reply:
I'll resolve this @sunker, but please reopen it if you think 10 characters wouldn't be ok.